Tests/SotoCoreTests/AWSClientTests.swift

```diff
@@ -576,52 +576,6 @@ class AWSClientTests: XCTestCase {
         }
     }
 
-    /// verify we are not calling the Retry handler when streaming a request
-    func testDontRetryStreamingRequests() async {
```

Comment:
Yes we don't need it anymore. Previously I couldn't run retry on streamed responses because they called a callback for each data chunk. If you retried the request you could be sending the same data twice. Now we return the header and an async sequence with the body data and this is no longer a concern.

You can see in AWSClient.swift line 451 the related code has been deleted